RELEASING.md

```diff
@@ -0,0 +1,63 @@
+# Releasing Kubeclient
+
+## Versioning
+Kubeclient release versioning follows [SemVer](https://semver.org/).
+At some point in time it is decided to release version x.y.z.
+Prerequisite: A [changelog](CHANGELOG.md) is generated & merged into the release branch.
+
+```bash
+RUBYGEMS_USERNAME=bob
+RELEASE_BRANCH="master"
+RELEASE_VERSION="x.y.z"
+GIT_REMOTE="origin"
+GIT_UPSTREAM="upstream"
+
+#
+# Install the release gem
+#
+gem install gem-release
+
+
+#
+# Checking out the release branch
+#
+git fetch $GIT_UPSTREAM
+git checkout $GIT_UPSTREAM/$RELEASE_BRANCH -b $RELEASE_BRANCH
```

Comment:
should also do `git status` / `git reset --hard` to ensure no uncommitted changes.